src/app/cart/templates/cart.html

```diff
@@ -32,7 +32,7 @@ <h5 ng-repeat="promotion in cart.promotions" class="text-danger">
                         <span class="pull-right">- {{promotion.Amount | currency}}</span></h5>
                 </div>
                 <div class="panel-footer">
-                    <button ng-if="cart.lineItems.Items.length > 0" type="button" ui-sref="checkout.shipping" class="btn btn-block btn-primary" ng-disabled="OrderLineItemForm.$invalid">Proceed to Checkout</button>
+                    <button ng-if="cart.lineItems.Items.length > 0" type="button" ng-click="cart.proceedToCheckout()" class="btn btn-block btn-primary" ng-disabled="OrderLineItemForm.$invalid">Proceed to Checkout</button>
                 </div>
```

Comment:
Why do you have an ng-click here? Looks like its just routing user to checkout.shipping state, so we can probably just keep the ui-sref?